skimage/transform/_thin_plate_splines.py

```diff
@@ -1,5 +1,6 @@
 import numpy as np
 from scipy import ndimage
+import scipy as sp
```

Comment:
I think @mkcor meant to replace the above line with the new import statement. You'll also need to update `ndimage.map_coordinates`, and `distance.cdist` below.